vm/devices/storage/nvme_spec/src/lib.rs

```diff
@@ -126,20 +131,33 @@ pub struct Aqa {
 }
 
 #[repr(C)]
-#[derive(Copy, Clone, Debug, AsBytes, FromBytes, FromZeroes, Inspect)]
+#[derive(Copy, Clone, Debug, AsBytes, FromBytes, FromZeroes, Inspect, Protobuf)]
+#[mesh(package = "underhill")]
```

Comment:
This is not the right approach. We shouldn't be saving our "interpreted" version of `Command`, we should just save the raw bytes. If we need to save a `Command`, use `ZeroCopyEncoding`, which will handle this automatically:

E.g.,
```rust
struct MySavedState {
    #[mesh(encoding = "mesh::payload::encoding::ZeroCopyEncoding")]
    command: Command,
}
```

Reply:
There was another comment from another reviewer to go this way instead.... okay, will revert / improve.

Reply:
More generally - its almost never the right move to add `mesh` as a dependency to a `_spec` or `_protocol` crate, when the actual spec/protocol being encoded is based on C-structs. If you need to take those base protocol types and encode them in protobuf (or JSON, or anything else really) - that should take the form of a type conversion / adapter layer _outside_ the `_spec/_protocol` crate itself.

This might be a good thing to encode in the Coding Conventions part of our guide.

Reply:
Agree, doesn't look good to me either. This change was a response to an earlier comment on why we need impl From<> for this type. Since @jstarks suggested a third way, by using ZeroCopyEncoding, I will try that.

Reply:
Refactored, marking as resolved. Thanks for `ZeroCopyEncoding` hint.